src/createLoader.js

```diff
@@ -1,8 +1,8 @@
 import { load as actionLoad } from './actions';
 
-export default (engine) => (store) => {
+export default (engine) => (store, ...engineOpts) => {
     const dispatchLoad = (state) => store.dispatch(actionLoad(state));
-    return engine.load().then((newState) => {
+    return engine.load(...engineOpts).then((newState) => {
```

Comment:
Currently I'm not a big fan of this feature but **IF** we're going to support runtime engine keys, only the key should be passed here to keep a clean and simple API. All other engine dependend options should be passed to the engine constructor itself. Like:

```js
import createEngine from 'my-redux-engine';

const engine = createEngine({ some: 'options' });
engine.load('key').then(....);
```

Also **IF** we add a feature like this, we need to add it to `engine.save` too!